README.md

```diff
@@ -23,13 +19,13 @@ jobs:
     runs-on: self-hosted
     steps:
       - name: Check out repository
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Run build
-        uses: matlab-actions/run-build@v1
+        uses: matlab-actions/run-build@v2
```

Comment:
Hi, @mw-kapilg. The version bump is not due to the new option; it's because of migrating to batch tokens. In other words, this release will include your work on build options as well as @davidbuzinski 's efforts on batch tokens.